README.md

```diff
@@ -16,6 +16,31 @@ This chart demonstrates 3 scenarios:
 2. `logs-by-label` collects all logs for pods with the label `rancher.logging.example/app: log-generator`
 3. `namespace-logs` collects all logs for the namespace this chart is deployed in.
 
+#### Resource Requests and Limits
+
+You can change `elasticsearch` resource requests and limits from the upstream chart by editing `values.yaml` as follows...
+
+```yaml
+elasticsearch:
+  resources:
+    requests:
+      cpu: "0"
+      memory: "0"
+    limits:
+      cpu: "0"
+      memory: "0"
+```
+
+#### Checking ElasticSearch
+
+You can see if `elasticsearch` is working by forwarding port `9200` and accessing the instance via `localhost`.

```

Comment:
Change this to "You can see if rancher logging is working by..."

Reply:
Agreed.